docs/configmap.md

```diff
@@ -0,0 +1,109 @@
+# Config Map
+
+## About
+The `configmap` which used to tune the `managed-upgrade-operator`. It has various configurable values.
+
+## How to use it
+
+### For OSD:
+The `configmap` will be synced to the clusters from the hive cluster via selectorsyncset. You can update the `configmap` via [`managed-cluster-config`](https://github.com/openshift/managed-cluster-config/tree/master/deploy/managed-upgrade-operator-config).
+
+### For ARO:
+TDB
```

Comment:
My personal preference is to leave out deployment-specific stuff in these docs and keep it generic (ie. "how to use it" from the perspective of a non-Red Hatter who won't know what Hive is), and maintain that in Source (as in, "The Source") documentation. But happy to hear what the rest of the team want to do here and go with the majority feeling.

Reply:
how about just mention the managed cluster config is the place for OSD? but not mention hive and syncset. 
for people easy to reference/locating the place where to update